nextflow.config

```diff
@@ -25,7 +25,7 @@
 
    // Reference genomes
    genomes_base = 's3://deepvariant-data/genomes'
-   genome = false
+   //genome = 'hg19' //if no fasta specified default of hg19 will be used (see main.nf)
```

Comment:
Hi @PhilPalmer - did you add all of your changes to this commit? The changes don't really match the commit message..

Cheers,

Phil